Dockerfiles/ebpf-builder.Dockerfile

```diff
@@ -24,6 +24,11 @@ RUN wget https://apt.llvm.org/llvm.sh && chmod +x llvm.sh && ./llvm.sh $CLANG_LL
 
 COPY --from=builder /usr/include/bpf /usr/include/bpf
 
+RUN \
+	wget --quiet https://github.com/tinygo-org/tinygo/releases/download/v0.30.0/tinygo_0.30.0_amd64.deb && \
+	echo abcef56b2ae04e27253df409b32d0b7abb5ae76ed493db75b2f80659cbf59363c85919836116780adcad051c652e991dc46fa6ae7cec31a4fa3bdb68d2123621 tinygo_0.30.0_amd64.deb | sha512sum -c && \
+	dpkg -i tinygo_0.30.0_amd64.deb
+
```

Comment:
`go install` does not handle pre-compiled package; it would necessarily recompile tinygo. Using a pre-compiled package from upstream is faster than recompiling. Also, tinygo uses a Makefile to be compiled. As far as I know, `go install` cannot call the Makefile: it just calls `go build`. Additionally, tinygo has build-dependencies on llvm. It seems excessive to install llvm in `golang:1.19`. In the end, I don't need to make any changes to tinygo, so picking up the upstream release is fine.